Rakefile

```diff
@@ -12,4 +12,6 @@ end
 
 task :default => 'deployinator:test:unit'
 
+task :character => 'deployinator:test:character'
```

Comment:
I'd prefer naming this something like 'test-character', or something that makes it clear that these are _tests_ that we are running (especially since I wasn't familiar with the term 'character tests' until very recently.

Reply:
`rake test:character`